app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Admin
+  class UsersController < ApplicationController
+  	before_filter :admin_authorize!
+  	def index
+  		@users = User.all
+    end		
```

Comment:
`end` at 6, 4 is not aligned with `def` at 4, 3<br>Trailing whitespace detected.